templates/default/systemd.erb

```diff
@@ -4,10 +4,11 @@ After=network.target
 Wants=network.target
 
 [Service]
+EnvironmentFile=-/etc/sysconfig/<%= @name %>
```

Comment:
EnvironmentFile is discouraged by systemd. You need to restart the service anyway if you want to use the updated variables.
https://github.com/systemd/systemd/issues/1082#issuecomment-136452207